app/services/green_lanes/determine_candidate_categories.rb

```diff
@@ -1,58 +1,54 @@
 module GreenLanes
-  class DetermineCategory
-    CAT_1 = 1
-    CAT_2 = 2
-    CAT_3 = 3
-
+  class DetermineCandidateCategories
```

Comment:
renamed DetermineCategory ->  DetermineCandidateCategories

Two reasons:
- clearer name, it differentiate more form `DetermineResultingCategories`
- it's plural, since it can return more than one categories.